internal/cmd/backup.go

```diff
@@ -398,6 +408,9 @@ func backupCreateCmdFunc(cmd *cobra.Command, args []string) (err error) {
 		}
 		return nil
 	})
+	if err != nil {
+		return err
+	}
```

Comment:
This was the substantive change. In #543 I refactored things to use `takeBackup`, which meant that the `return`s that moved into `takeBackup` no longer interrupted execution. I missed returning `err` here because I assumed that it was at the bottom; `ineffassign` didn't yell because of the `defer` blocks that take a pointer to `err`.